CHANGELOG.unreleased.md

```diff
@@ -20,6 +20,7 @@
 - [SDN Controller] Do not choose physical PIF without IP configuration for tunnels. (PR [#4319](https://github.com/vatesfr/xen-orchestra/pull/4319))
 - [Xen servers] Fix `no connection found for object` error if pool master is reinstalled [#4299](https://github.com/vatesfr/xen-orchestra/issues/4299) (PR [#4302](https://github.com/vatesfr/xen-orchestra/pull/4302))
 - [Backup-ng/restore] Display correct size for full VM backup [#4316](https://github.com/vatesfr/xen-orchestra/issues/4316) (PR [#4332](https://github.com/vatesfr/xen-orchestra/pull/4332))
+- [VM/tab-advanced] Fix CPUs and max CPUs edition (PR [#4337](https://github.com/vatesfr/xen-orchestra/pull/4337))
```

Comment:
What do you fix in particularly in CPUs ?

Reply:
I fixed their edition in the `VM/tab-advanced` UI.

Reply:
Then this should be: 
- [VM/tab-advanced] Fix CPUs edition (PR [#4337](https://github.com/vatesfr/xen-orchestra/pull/4337))

Reply:
Why it should be `Fix CPUs edition` instead of `Fix CPUs and max CPUs edition` ? Given that i fixed the two things: the VM CPUs and the VM max CPUs edition.

Reply:
What did you fix in `VM CPUs` ?

Reply:
Their edition.